services/network_config_manager/src/main/java/com/futurewei/alcor/netwconfigmanager/server/grpc/GoalStateProvisionerServer.java

```diff
@@ -32,7 +32,7 @@
 //    private GoalStateClient grpcGoalStateClient;
 
     public GoalStateProvisionerServer() {
-        this.port = 50010;
+        this.port = 9016;
```

Comment:
@xieus Should we put this value in the application.properties?

Reply:
That is right. I will add it in next PR.